src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -161,14 +163,97 @@ public void cannotUpdateCirculationRulesWithLostItemPolicyId() {
       overdueFinePoliciesFixture.facultyStandard().getId().toString(),
       UUID.randomUUID().toString());
 
-    Response response = circulationRulesFixture
-      .attemptUpdateCirculationRules(rule, "i");
+    Response response = circulationRulesFixture.attemptUpdateCirculationRules(rule);
 
     assertThat(response.getStatusCode(), is(422));
     assertThat(response.getJson().getString("message"),
       is("The policy i does not exist"));
   }
 
+  @Test
+  public void canUpdateCirculationRulesWithTwentyExistingLoanPolicies() {
+
+    Set<UUID> loanPolicyIds = getSetOfPolicyIds(20);
+    addPoliciesToLoanPolicyFixture(loanPolicyIds);
+
+    loanPolicyIds.forEach(loanPolicyId -> {
+      String rule = circulationRulesFixture.soleFallbackPolicyRule(
+        loanPolicyId.toString(), requestPoliciesFixture.allowAllRequestPolicy().getId().toString(),
+        noticePoliciesFixture.activeNotice().getId().toString(),
+        overdueFinePoliciesFixture.facultyStandard().getId().toString(),
+        lostItemFeePoliciesFixture.facultyStandard().getId().toString());
```

Comment:
It is difficult to understand how the rule is created in this and the following tests.
Can you remove `toString()` from all passed arguments?
Can you add a line break after `loanPolicyId.toString(),`?

Reply:
updated, but I had to extend visibility for method soleFallbackPolicyRule(UUID loanPolicyId, UUID requestPolicyId, UUID noticePolicyId, UUID overdueFinePolicyId, UUID lostItemFeePolicyId). I have changed it to public. 
Public method with the same name takes only String parameters.